apps/src/templates/instructions/TopInstructionsCSP.jsx

```diff
@@ -62,6 +64,29 @@ var styles = {
     height: HEADER_HEIGHT,
     lineHeight: HEADER_HEIGHT + 'px'
   },
+  helpTab: {
+    float: 'left',
+    paddingTop: "6px",
+    paddingLeft: "30px",
```

Comment:
For most properties in react, you can specify a number rather than a string and it will auto-append 'px'. Using the numerical version is generally preferred in our codebase.

That said, you do need to watch out for the few exceptions where React does not auto-px. The main one of these that I know of is `line-height`.

Reply:
Still using px for padding Top/Left above